src/components/avatar.js

```diff
@@ -26,7 +26,10 @@ export default function AlignItemsList() {
         <List className={classes.root}>
             <ListItem alignItems="flex-start">
                 <ListItemAvatar>
-                    <Avatar alt="Isisphinx" src="src/images/face.png" />
```

Comment:
il faut importer l'image : 
https://www.gatsbyjs.com/docs/how-to/images-and-media/importing-assets-into-files/